filebeat/docs/getting-started.asciidoc

```diff
@@ -56,20 +56,27 @@ include::{libbeat-dir}/tab-widgets/install-widget.asciidoc[]
 include::{libbeat-dir}/shared/connecting-to-es.asciidoc[]
 
 [float]
-[[enable-modules]]
-=== Step 3: Enable and configure data collection modules
+[[collect-log-data]]
+=== Step 3: Collect log data
+
+There are three ways to collect log data with {beatname_uc}:
 
-{beatname_uc} uses modules to collect and parse log data.
+* Data collection modules -- simplify the collection, parsing,
+and visualization of common log formats
+* ECS loggers -- plugins that structure and format
```

Comment:
```suggestion
* ECS loggers -- structure and format
```

Reply:
Edited to maintain parallelism. If having "plugins" in the description is necessary, can you say ECS logger plugins?